pyvista/core/filters/data_set.py

```diff
@@ -4722,6 +4722,43 @@ def merge(
                 raise TypeError(f"Mesh type {type(self)} cannot be overridden by output.")
         return merged
 
+    def merge_points(self, tolerance=0.0, progress_bar=False):
+        """Merge points of this dataset.
+
+        Parameters
+        ----------
+        tolerance : float, default: 0.0
+            The absolute tolerance to use to find coincident points.
+
+        progress_bar : bool, default: False
+            Display a progress bar to indicate progress.
+
+        Returns
+        -------
+        UnstructuredGrid
+            Dataset with merged points.
+
+        See Also
+        --------
+        PolyDataFilters.clean : Preferable with :class:`pyvista.PolyData`.
+
+        Examples
+        --------
+        Create a structured cylinder and merge interior faces. This prevents
+        additional "internal" faces when extracting the surface of the
+        cylinder.
+
+        >>> import pyvista as pv
+        >>> cyl = pv.CylinderStructured(radius=[0, 0.1, 0.2])
+        >>> cyl.n_points
+        990
+        >>> cyl_merged = cyl.merge_points()
+        >>> cyl_merged.n_points
+        670
+
+        """
+        return self.merge(tolerance=tolerance, progress_bar=progress_bar)
```

Comment:
I agree that this name is better, but what about deprecating the `merge` method and replacing it with `merge_points`?
In DataSet, I think only `merge_points` is sufficient.

Reply:
`merge` is still necessary in order to merge multiple datasets. `merge_points` is just for merging points of the dataset itself.

Reply:
Got it. Thanks!